### DISCUSSION TITLE
⌘ or ⌥ + Drag & Drop from the side bar to copy a block

### DISCUSSION THREAD

**Discussion by Limezy** [athensresearch/athens#965] [Feature Requests]
- Right now, drag & dropping a block from the side bar to the main page will move it.
- ⇧ + drag & dropping a block from the side bar will create a block quote.

I would like ⌘ or ⌥ + drag & dropping to copy the block. 
⌥ sounds better because it's already the natural keystroke to duplicate, at least for OSX users.

**Comment by Limezy** [comment]
Sorry, just saw that it was referenced as an issue here https://github.com/athensresearch/athens/issues/572
I don't know how to close the discussion

**Reply by jefftangx** [reply]
Just to clarify:

- [x] copy block ref
- [x] move block
- [ ] copy block <~ this is what you want @Limezy

**Reply by Limezy** [reply]
Yes, exactly !
